server/bin/pbench-server-prep-shim-002.py

```diff
@@ -166,7 +166,7 @@ def process_tb(config, logger, receive_dir, qdir_md5, duplicates, errors):
         # First, copy the small .md5 file to the destination. That way, if
         # that operation fails it will fail quickly since the file is small.
         try:
-            shutil.copy2(tbmd5, dest)
+            shutil.copy(tbmd5, dest)
```

Comment:
Does the `shutil.move()` call on l.190 do the right thing for the tarball? IIUC, it does not and we need to do something there. I wonder if the thing to do is leave the `shutil.copy2()` alone (again IIUC, that's the equivalent of `cp -a` in the shell) and do explicit `restorecon` calls (using the `selinux` module from the `libselinux-python3` RPM package) to set the SELinux context on the tar ball and md5 files.

Reply:
seems fair

Reply:
How should I import SELinux in code? when I try the python3 command line, it is working after importing. In code, it is saying no module named "selinux".

Reply:
Do we need to define in PATH?

Reply:
If you have the `libeslinux-python3` RPM installed, you should have an `selinux` python3 module in `/usr/lib64/python3.6/site-packages/selinux` (on RHEL7). Then you should be able to
```
import selinux
...
selinux.restorecon(<path>)
...
```
Does that not work?

Reply:
I am using RHEL8 and It is failing test cases.

Reply:
I believe this comment is now resolved.